### PR TITLE
[RU] player turns

### PR DESCRIPTION
**Pull request by ohgodnotagain**
ru-10
ru-11
ru-12

### CODE REVIEW

**Comment by qwrtln** [comment]
You know what, maybe let's merge the PR which fixes fonts first. Those titles are awful.

File: translations/player_turns.tex/ru.po [hunk 1]
```diff
@@ -77,6 +77,18 @@ msgid ""
 "Once all players have spent all their Movement Points and do not wish to use any further Town or Morale Actions, the current Round is over.\n"
 "\n"
 msgstr ""
+"В начале вашего хода обновите карты в руке следующим образом:\n"
+"\\begin{itemize}\n"
+"  \\item сбросьте любое количество карт с руки.\n"
+"Если количество карт в руке больше лимита \\includesvg[height=10px]{\\svgs/hand.svg}, сбросьте лишние карты.\n"
+"  \\item доберите недостающее до лимита руки количество карт.\n"
+"  \\item выполните все указания из секции \"в начале вашего хода...\".\n"
```
Comment (DarthGandalf) [inline translations/player_turns.tex/ru.po:85]:
```suggestion
"  \\item выполните все эффекты вида «в начале вашего хода...».\n"
```

File: translations/player_turns.tex/ru.po [hunk 2]
```diff
@@ -160,6 +201,19 @@ msgid ""
 "\\end{itemize}\n"
 "\n"
 msgstr ""
+"\\subsection*{Действия Города\\index{Town Actions}}\n"
+"Вы можете выполнять каждое из описанных ниже действий города \\textbf{1 раз в раунд}.\n"
+"Эти действия могут быть выполнены в любой момент хода любого игрока, за исключением сражения или если такое действие воспрепятствует другому действию.\n"
+"For example, you cannot draw Spell cards simultaneously with the Spell Book Token.\\par\n"
+"Когда игрок заявляет о намерении начать сражение, вы можете выполнить любое из действий города перед выполнением \\hyperlink{Combatsetup}{подготовки к бою.}\\par\n"
+"После выполнения действия города, переверните соответствующий жетон на карте города неактивной стороной вверх.\n"
```
Comment (DarthGandalf) [inline translations/player_turns.tex/ru.po:209]:
```suggestion
"После выполнения действия города, переверните соответствующий жетон на карте города лицом вниз.\n"
```

File: translations/player_turns.tex/ru.po [hunk 3]
```diff
@@ -160,6 +201,19 @@ msgid ""
...
+"Вы не можете выполнить это действие еще раз до начала следующего раунда.\n"
```
Comment (DarthGandalf) [inline translations/player_turns.tex/ru.po:210]:
```suggestion
"Вы не можете выполнить это действие ещё раз до начала следующего раунда.\n"
```

File: translations/player_turns.tex/ru.po [hunk 4]
```diff
@@ -181,6 +235,21 @@ msgid ""
...
+"У вас может быть только один такой жетон.\n"
+"Если вы собираетесь получить второй такой жетон, вы можете сразу применить первый перед получением второго.\n"
+"Жетон положительной морали может быть использован в любое время для выполнения любого из действий:\n"
+"\\begin{itemize}\n"
+"  \\item взять карту из вашей колоды.\n"
+"  \\item сбросить любое количество карт и взять такое же количество.\n"
+"  \\item перебросить любой кубик.\n"
+"\\end{itemize}\n"
+"Если вы теряете мораль, сбросьте жетон положительной морали\\includesvg[height=10px]{\\svgs/morale_positive.svg} если он у вас есть, или возьмите жетон отрицательной морали \\index{Negative Morale}\\includesvg[height=10px]{\\svgs/morale_negative.svg}.\n"
+"В свою очередь, получение положительной морали отменяет отрицательную мораль при ее наличии.\n"
```
Comment (DarthGandalf) [inline translations/player_turns.tex/ru.po:250]:
```suggestion
"В свою очередь, получение положительной морали отменяет отрицательную мораль при её наличии.\n"
```

File: translations/player_turns.tex/ru.po [hunk 5]
```diff
@@ -191,6 +260,10 @@ msgid ""
 "}\n"
 "\n"
 msgstr ""
+"\\note{5}{Фракция Некрополиса \\includesvg[height=10px]{\\svgs/necro-note.svg} игнорирует все эффекты морали.\n"
+"  Ее герои не могут получать или терять мораль.\n"
```
Comment (DarthGandalf) [inline translations/player_turns.tex/ru.po:264]:
```suggestion
"  Некрополис ни при каких обстоятельствах не может получать или терять мораль.\n"
```

File: translations/player_turns.tex/ru.po [hunk 6]
```diff
@@ -223,6 +298,14 @@ msgid ""
 "As Alice announces her intent to start Combat, both players still have an opportunity to perform Town Actions.}\\par\n"
 "\n"
 msgstr ""
+"\\textit{Алиса, играющая за рыцаря Катерину, начинает свой ход.\n"
+"У неё есть 3 карты на руке с предыдущего раунда, и она решает сбросить 2 из них перед набором карт для своей колоды, солгасно её лимиту руки \\includesvg[height=10px]{\\svgs/hand.svg}.\n"
```
Comment (DarthGandalf) [inline translations/player_turns.tex/ru.po:302]:
```suggestion
"У неё есть 3 карты на руке с предыдущего раунда, и она решает сбросить 2 из них перед набором карт для своей колоды, согласно её лимиту руки \\includesvg[height=10px]{\\svgs/hand.svg}.\n"
```

File: translations/player_turns.tex/ru.po [hunk 7]
```diff
@@ -223,6 +298,14 @@ msgid ""
...
+"Так как уровеь её основного героя 3, то лимит равен 5, она набирает 4 карты после сброса (см. \\hyperlink{Level}{Уровни Эффектов}).}\\par\n"
```
Comment (DarthGandalf) [inline translations/player_turns.tex/ru.po:303]:
```suggestion
"Так как уровень её основного героя 3, то лимит равен 5, она набирает 4 карты после сброса (см. \\hyperlink{Level}{Уровни Эффектов}).}\\par\n"
```

File: translations/player_turns.tex/ru.po [hunk 8]
```diff
@@ -223,6 +298,14 @@ msgid ""
...
+"\\textit{После она использует жетон строительства для постройки Жилища \\includesvg[height=10px]{\\svgs/golden.svg} и жетон популяции для \\hyperlink{Units}{найма} отряда Чемпионов \\includesvg[height=10px]{\\svgs/golden.svg}.\n"
+"Она может это сделать, так как имела ранее построенные Жилища предыдущих уровней (\\includesvg[height=10px]{\\svgs/bronze.svg} и \\includesvg[height=10px]{\\svgs/silver.svg}) и имела достаточно \\hyperlink{Resources}{ресурсов} для постройки Жилищ и найма Чемпионов.}\\par\n"
+"\\textit{Теперь, готовая к предстоящей битве, она использует ОП для передвижения её главного героя в соседнее поле, оккупированное некромантом Сандро, вражеским героем под контролем Боба.\n"
+"Так как Алиса объявляет намерение начать сражение, оба игрока все еще имеют возможность выполнить действия города.}\\par\n"
```
Comment (DarthGandalf) [inline translations/player_turns.tex/ru.po:307]:
```suggestion
"Так как Алиса объявляет намерение начать сражение, оба игрока всё ещё имеют возможность выполнить действия города.}\\par\n"
```

**Review by DarthGandalf — CHANGES_REQUESTED** [review]

**Review by DarthGandalf — APPROVED** [review]

**Review by qwrtln — APPROVED** [review]

**Pull request merged**